frontend/packages/dev-console/integration-tests/views/pipeline.view.ts

```diff
@@ -13,8 +13,37 @@ export const pipelineTable = element(
 export const pipelineTableBody = element(
   by.className('ReactVirtualized__VirtualGrid__innerScrollContainer'),
 );
+export const samplePipelineElement = element(by.css('[data-test-id="sample-app-pipeline"]'));
 export const pipelineGraph = $('.odc-pipeline-vis-graph__stages');
 export const pipelineOverviewName = element(by.css('[data-test-id="resource-title"]'));
+export const pipelineFilter = element(by.className('pf-c-form-control co-text-filter'));
```

Comment:
Grabbing items by very common classes is a bad idea. Assuming this doesn't just grab every instance in the document, you're going to end up with a lot of failures for unknown reasons purely based on the order of rendering.